pkg/gadgets/trace/exec/tracer/bpf/execsnoop.bpf.c

```diff
@@ -84,6 +89,12 @@ int ig_execve_e(struct trace_event_raw_sys_enter* ctx)
 	event->args_size = 0;
 	event->mntns_id = mntns_id;
 
+	if (get_cwd) {
+		fs = BPF_CORE_READ(task, fs);
+		cwd = get_path_str(__builtin_preserve_access_index(&fs->pwd));
```

Comment:
Why do we need __builtin_preserve_access_index?

Reply:
I just wanted to make it explicit a relocation happens there, but it's not needed since we're using vmlinux.h. I removed it.